README.md

```
@@ -1,2 +1,46 @@
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=soat-fiap_bmb.production&metric=coverage)](https://sonarcloud.io/summary/new_code?id=soat-fiap_bmb.production)
+
+## Event Subscriptions
+
+### OrderCreated Event
+- Subscribes to `OrderCreated` events.
+- Adds the order to the orders list using the order ID as the key.
+
+### OrderPaymentConfirmed Event
+- Subscribes to `OrderPaymentConfirmed` events.
+- If the orders list contains the received OrderId:
+    - Adds `TrackingCode:OrderId` to the `Received` queue.
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)